assets/src/styles/campaigns/base/_fonts.scss

```diff
@@ -1,37 +0,0 @@
-@font-face {
```

Comment:
This one was still depended on in the editor to load the fonts also for new themes. Not sure if re-adding it is best, as we may want to just look into loading all fonts a theme includes on the fly.